tensorflow/core/kernels/rnn/blas_gemm.h

```diff
@@ -25,6 +25,10 @@ limitations under the License.
 #include "tsl/framework/contraction/eigen_contraction_kernel.h"
 #endif
 
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
```

Comment:
Why is this header added here when not used in this file?  We need each source file to directly include the headers it explicitly requires.

Reply:
Fixed